tests/drivers/dma/chan_blen_transfer/src/test_dma.c

```diff
@@ -23,7 +23,18 @@
 #include <drivers/dma.h>
 #include <ztest.h>
 
+#if defined(CONFIG_SOC_FAMILY_STM32)
+#if defined(CONFIG_SOC_SERIES_STM32F2X) || \
```

Comment:
Would it make sense to use matching driver variant config symbol here (`CONFIG_DMA_STM32_Vx`)

Reply:
Right

Reply:
Unrelated nit, but could have `#ifdef CONFIG_FOO` instead of `#if defined(CONFIG_FOO)` when there's just one. I don't know if we just do it for consistency, but it looks a bit odd coming to Zephyr from the outside

Reply:
> Unrelated nit, but could have `#ifdef CONFIG_FOO` instead of `#if defined(CONFIG_FOO)` when there's just one. I don't know if we just do it for consistency, but it looks a bit odd coming to Zephyr from the outside

No guidelines here so I think they are all ok. usually I would use what others are using. In Zephyr I saw many examples of using #if defined(CONFIG_FOO) so I'm using it. :thinking:

Reply:
Think it's more that it's just getting copied around by people though. Fight the flow. ;)